src/legacy/core_plugins/metrics/public/components/aggs/moving_average.js

```diff
@@ -34,58 +33,82 @@ import {
   EuiComboBox,
   EuiSpacer,
   EuiFormRow,
-  EuiCode,
-  EuiTextArea,
+  EuiFieldNumber,
 } from '@elastic/eui';
-import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { MODEL_TYPES } from '../../../common/model_options';
 
-const MovingAverageAggUi = props => {
-  const { siblings, intl } = props;
-  const defaults = {
-    settings: '',
-    minimize: 0,
-    window: '',
-    model: 'simple'
-  };
-  const model = { ...defaults, ...props.model };
-  const handleChange = createChangeHandler(props.onChange, model);
-  const handleSelectChange = createSelectHandler(handleChange);
-  const handleTextChange = createTextHandler(handleChange);
-  const handleNumberChange = createNumberHandler(handleChange);
+const DEFAULTS = {
+  model_type: MODEL_TYPES.UNWEIGHTED,
+  alpha: 0,
+  beta: 0,
+  gamma: 0,
+  period: 1,
+  multiplicative: true,
+  window: 10,
+};
+
+const isProperWindowValue = ({ window, period }) => window > (period * 2);
```

Comment:
From the docs https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-movfn-aggregation.html#_cold_start_2
> your window must always be at least twice the size of your period.

Reply:
Changed. Thanks